k8s/helmfile/helmfile.yaml

```diff
@@ -217,6 +222,12 @@ releases:
     version: 5.0.0
     <<: *default_release
 
+  - name: istio-service-mesh
+    namespace: istio-system
+    installed: {{ ne .Environment.Name "production" | toYaml }}
+    chart: ./../../charts/istio-service-mesh
```

Comment:
This is totally fine but we should be slightly aware of hassle of putting things here; it can make rolling updates in the future hard since `main` of this repo is the truth we build all environments from; we should be aware that if we end up wanting a new version of this chart in one environment but not the others we'll need to move it to the charts repo (or duplicate it) etc.

Reply:
Good point, didn't think of this. There probably is no way to pass values to `Chart.yaml` itself?